tests/FSharp.Compiler.ComponentTests/ErrorMessages/ConstructorTests.fs

```diff
@@ -26,6 +26,7 @@ doSomething {Record.field1=0; field2=0}
 type Person = { Name : string; Age : int; City : string }
 let x = { Name = "Isaac", Age = 21, City = "London" }
         """
+        |> withLangVersion50
```

Comment:
Is there a specific reason this needed langversion 5.0?

Reply:
Hey @dsyme , indeed yes and amazingly I can even remember why ... the test verified the behaviour without preview [*Right Attribute Module Abbreviation without preview (compile)*`].  I.e. the old F#5.0 behaviour, so ... this the test became with net50 [*Right Attribute Module Abbreviation with version 5.0 (compile)*] instead

Reply:
This... isn't the test you are referring to.